js/util/script_detection.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const isVerticalWritingModeRegExp = new RegExp([
+    '[ᄀ-ᇿ]',
+    '[가-힣]',
+    '[ㄱ-ㆎ]',
+    '[一-鿌]',
+    '[㐀-䶵]',
+    // eslint-disable-next-line no-irregular-whitespace
+    '[　-〿]',
```

Comment:
This matches everything from U+0020 (space) to U+303F, including the entire ASCII alphabet.